src/styles/index.ts

```diff
@@ -5277,6 +5277,10 @@ const styles = (theme: ThemeColors) =>
             borderColor: theme.border,
             padding: 16,
         },
+
+        outsetShadow: {
```

Comment:
I feel like we could have a better name for this? It is true that it is using a shadow to make the effect happen, but it seems more appropriate to call this something like "selectionOuterBorder" or something? Thoughts?